without_std.rs

```diff
@@ -0,0 +1,53 @@
+extern "C" {
```

Comment:
What happens when something that uses the crate in `wasm`, also defines a `GlobalAllocator`? I would assume that we get a linker error?